website/events/models/event.py

```diff
@@ -486,9 +486,9 @@ def delete(self, using=None, keep_parents=False):
             self.registration_reminder is not None
             and not self.registration_reminder.sent
         ):
-            collector.add([self.registration_reminder])
+            collector.collect([self.registration_reminder], keep_parents=keep_parents)
         if self.start_reminder is not None and not self.start_reminder.sent:
-            collector.add([self.start_reminder])
+            collector.collect([self.start_reminder], keep_parents=keep_parents)
         if self.has_food_event:
             collector.add([self.food_event])
```

Comment:
Should this then also be collect?

Reply:
I also checked that. It's not necessary, it works like this. But I can of course change it for consistency